src/dash/vo/DescriptorType.js

```diff
@@ -44,6 +46,16 @@ class DescriptorType {
             this.schemeIdUri = data.schemeIdUri ? data.schemeIdUri : null;
             this.value = data.value ? data.value : null;
             this.id = data.id ? data.id : null;
+            // Only add the DVB extensions if they exist
+            if (data[DashConstants.DVB_URL]) {
+                this.dvb_url = data[DashConstants.DVB_URL]
```

Comment:
I would like to stick to the camelCase style here instead of using underscore style. Although `dvb_` is probably used as a prefix here, I suggest we use `dvbUrl` , `dvbMimeType` and `dvbFontFamily`.

Reply:
I chose that naming based on what had already been done in [BaseUrl.js](https://github.com/Dash-Industry-Forum/dash.js/blob/427b2e790f670d449d532ef60b003bad8ed5ec92/src/dash/vo/BaseURL.js#L45). Do you think it's still best to make the change given that?

Reply:
I've changed these dvb extensions to use camelCase [here](https://github.com/Dash-Industry-Forum/dash.js/pull/4338/commits/eecb7d04d33ec9cf1f57e25345ed44ed5f5bc3ed). Let me have a crack at the `BaseURL.js` properties (in a separate PR) as I've brought about this change and then if you review it we should catch everything 👍

Reply:
See #4362